packages/backend/src/engine/custom_outputs_accumulator.test.ts

```diff
@@ -104,7 +104,9 @@ void describe('Custom outputs accumulator', () => {
     );
 
     accumulator.addOutput({
-      aws_user_pools_id: 'some_user_pool_id',
+      auth: {
+        user_pool_id: 'some_user_pool_id',
+      },
```

Comment:
not sure where to leave this comment.
we should add validation: if somebody `addOutput({ version: 1, ...})` and then generates config with `--version 2` then this should throw runtime error somehow.

Reply:
also attempt to use `backend.addOutput({ version: 1, ...}); backend.addOutput({ version: 2, ...})` should also throw.

Reply:
Added both validations.